## Introduction {.page_break_before}

Proteomics is the large-scale study of protein structure and function. 
Proteins are translated from mRNAs that are transcribed from the genome. 
Although the genome encodes potential cellular functions and states, the study of proteins in all their forms is necessary to truly understand biology. 

Currently, proteomics can be performed with various methods. 
Alternative methods based on affinity interactions of antibodies or DNA aptamers have been developed, namely Somascan and Olink. 
There are also nascent methods such as nanopores that are under development and not yet applicable to whole proteomes.
Another approach uses parallel immobilization of peptides with total internal reflection microscopy and sequential edman degradation [@DOI:10.1038/nbt.4278].
However, by far the most common method for proteomics is based on mass spectrometry with liquid chromatography.

Modern proteomics started around the year 1990 with the introduction of soft ionization methods that enabled, for the first time, transfer of large biomolecules into the gas phase without destroying them [@DOI:10.1126/science.2675315; @DOI:10.1002/rcm.1290020802]. 
Shortly afterward, the first computer algorithm for matching peptides to a database was introduced [@PMID:24226387]. 
Another major milestone that allowed identification of over 1000 proteins were actually improvements to chromatography [@DOI:10.1021/ac010617e]. 
As the volume of data exploded, methods for statistical analysis transitioned use from the wild west to modern informatics based on statistical models [@DOI:10.1021/ac0341261] and the false discovery rate [@DOI:https://doi.org/10.1038/nmeth1019]. 
<!-- Todo: figure 1: major milestones in proteomics technology since 1990 -->

Two strategies of mass spectrometry-based proteomics differ fundamentally by whether proteins are cleaved into peptides before analysis: "top-down" and "bottom-up". 
Bottom-up proteomics (also refered to as shotgun proteomics) is defined by the hydrolysis of proteins into peptide pieces [@DOI:10.1038/nature19949]. 
Therefore, bottom-up proteomics does not actually measure proteins, but must infer their presence [@DOI:10.1021/ac0341261]. 
Sometimes proteins are inferred from only one peptide sequence representing a small fraction of the total protein sequence predicted from the genome. 
In contrast, top-down proteomics attempts to measure all proteins intact [@DOI:10.1039/C9MO00154A]. 
The potential benefit of top-down proteomics is the ability to measure proteoforms [@DOI:10.1126/science.aat1884]. 
However, due to myriad analytical challenges, the depth of protein coverage that is achievable by top-down proteomics is generally less than that of bottom-up proteomics [@DOI:10.1021/jasms.1c00099; @DOI:10.1021/acs.analchem.7b04747]. 

In this tutorial we focus on the bottom-up proteomics workflow. 
The most common version of this workflow is generally comprised of the following steps. 
First, proteins in a biological sample must be extracted. 
Usually this is done by denaturing and solubilizing the proteins while mechanically disrupting DNA and tissue to minimize interference in the analysis procedures. 
Next, proteins are hydrolyzed into peptides, usually using a protease like trypsin, which produces basic c-terminal amino acids to aid in fragment ion series production during tandem mass spectrometry. 
Peptides from proteome hydrolysis must be purified,; most often this is done with reversed phase chromatography cartridges or tips. 
The peptides are then almost always separated by liquid chromatography before they are ionized and introduced into a mass spectrometer, although recent reports describe LC-free proteomics by direct infusion [@DOI:10.1038/s41592-020-00999-z, @DOI:10.1021/acs.analchem.2c02249, @DOI:10.1101/2023.06.26.546628]. 
The mass spectrometer then collects precursor and fragment ion data from those peptides. 
The data analysis is usually the rate limiting step.
Peptides must be identified, and proteins are inferred, and quantities are assigned. 
Changes in proteins across conditions are determined with statistical tests, and results must be interpreted in the context of the relevant biology.

There are many variations to this workflow. 
The wide variety of experimental goals that are achievable with proteomics technology leads to a wide variety of potential proteomics workflows. 
Even choice is important, and every choice will affect the results. 
In this tutorial, we cover all the required steps in detail to serve as a comprehensive overview for new proteomics practioners. 
There are 16 sections in total:

1. Biochemistry basics
2. Types of experiments
3. Protein extraction
4. Proteolysis
5. Peptide Quantification Methods
6. Enrichments
7. Peptide purification
8. Liquid Chromatography
9. Peptide Ionization
10. Mass Spectrometry
11. Peptide Fragmentation (MS/MS)
12. Data Acquisition
13. Raw Data Analysis
14. Protein Databases
15. Biological Interpretation
16. Orthogonal Validation Experiments

