## Biological Interpretation {.page_break_before}

As described in previous sections, quantitative proteomics experiments reveal protein abundance changes. 
After statistical data analysis, the typical output is a list of differentially expressed proteins that can contain hundreds of hits. 
The next step is how to interpret the results in biological terms. 
Often, it is tempting to focus on proteins with the most extreme fold changes. 
In this case, the assumption is that the more significant the fold change (in either direction, up- or down-regulation), the higher the impact of those proteins on cellular behavior. 
Alas, this assumption is not always valid. 
Another approach is to start googling for every protein in the list to uncover groups that function together, maybe in protein complexes or signaling pathways. 
This strategy is time-consuming and cumbersome, and it is almost impossible to capture all reported interactions from the wealth of the literature. 

Constructing a protein network

A network is a representation of the relations between objects. 
Nodes are the entities of the network (users of a social platform, train stations, proteins), while edges are the connections between them (friendship, routes, and protein interactions, respectively). 
In the case of protein-protein interactions, the evidence for the functional associations between proteins can be obtained experimentally. 
For example, co-immunoprecipitation, crosslinking, and proximity labeling can be used to reveal physical interactions. 
The data is presented in a table with nodes and edges (e.g., "protein A interacts with protein B") from which the network can be constructed. 
A considerable wealth of protein-protein association data is stored in free databases like IntAct, which contain interactions derived from literature curation or direct user submissions. 
Protein interactions can also be predicted by classifiers that consider many features, like orthology and co-localization, to produce a posterior odds ratio of interaction. 
Finally, large repositories like STRING (Search Tool for the Retrieval of Interacting Genes/Proteins) collect and integrate protein-protein interaction data from several databases. 
It also provides a web-based interface to survey the data, and users only have to feed a search box with the identifiers of the protein(s) of interest. 
STRING will retrieve the network and show the evidence supporting each interaction. 
Importantly, these databases do not indicate the direction of the interaction, so they produce undirected networs. 
If edges have directions (e.g., A influences B and not vice versa), then the network is directed. Signaling pathways are examples of directed graphs. 

Network analysis

Network analysis is a group of techniques that explore and investigate the network, yielding valuable knowledge about its structure and unveiling key players regulating the flow of information. 
One of the first steps in network analysis relates to centrality measurements. 
Centralities are indicators of the relative importance of a node corresponding to its position in the network, and each centrality measure provides new insights to interpret the data in new ways.

Degree centrality
  
The degree of a node measures the number of edges incident to that node. 
Nodes with a high degree interact with many other nodes, called first neighbors. 
In particular, the node degree distribution in protein networks is highly skewed, with most nodes having a low degree and a few having high degrees, known as hubs.
Hubs are usually regulatory proteins, being notable examples oncogenes and transcription factors. 
Moreover, hubs are attractive targets for directed interventions, as their alteration has a profound effect on the stability of the network. 

Closeness centrality

The route from one node to another is a path, and the shortest path is the one connecting them in the least amount of steps. 
Closeness centrality is the inverse of the average length of a node's shortest paths to all other nodes in the network. 
Nodes with a high closeness score have the shortest distances to all the others, so closeness centrality calculations detect nodes that can spread information very efficiently, as they are in a better position in the network for this task.

Betweenness centrality

This centrality index is related to the amount of shortest paths transversing a node. 
Nodes with a high betweenness centrality usually bridge different parts of the network and strongly influence the flow of information, as they lie in communication paths. 
These connector hubs (or bottlenecks) are also interesting for followâ€“up experiments because their removal can disconnect different regions of the network.

Centrality measurements add new layers of information and allow for ranking differentially expressed proteins apart from their fold-change in abundance. 
Figure X depicts a simple network consisting of proteins A to L, with A having the highest fold change (10) and L the lowest (2). 
In Panel A, the fill color for the nodes indicates this metric, where it can be easily seen that A stands out. 
However, protein A is a peripheral protein, only interacting with B. 
In Panel B, nodes are colored according to node degree. 
Clearly, protein F has the highest number of interactions and is also the closest to all other nodes, which can be appreciated when nodes are colored according to closeness centrality (Panel C). 
On the other hand, protein G acts as a bridge between two regions of the network and thus, has the highest betweenness centrality (Panel D). 
Except for fold change, node A has the lowest indices, and it will be up to the researcher to decide whether this protein warrants further examination.

![Caption for the example figure.](https://github.com/ger225/proteomics-tutorial/blob/94baf66723b93100b1697def2e6fe3c2f6dfb295/content/images/Simple%20network.png){#fig:simple-network}

Network clustering

In the small network presented in Figure X, two groups of densely connected nodes exist. 
This topology suggests that these communities (or "clusters") work together or participate in a protein complex. 
Dividing a network into clusters helps identify underlying relationships among nodes, which is especially useful in large networks. 
In a broad sense, network clustering groups nodes according to a topological property, generally interconnectedness. 
There are many network clustering algorithms, each with its own merits and approaches. 
The MCL (Markov CLustering) algorithm is suitable for protein networks in most situations. 
On the other hand, the Molecular COmplex DEtection (MCODE) algorithm helps detect very densely connected nodes, thus unveiling protein complexes. 
In this regard, network clustering is useful for tentatively assigning the function of an uncharacterized protein. 
If the protein appears in a cluster, its function should be closely related to the cluster members, a principle known as "guilty by association."

Enrichment analysis
