## Protein Knowledge Bases {.page_break_before}

### PeptideAtlas and SRMAtlas
For the vast majority of shared raw mass spectrometry data, none of the data was compared in toto to derive at a knowledge base of all detectable proteins in an organism.
In addition, given the vast array of software for MS data analysis, the results are not directly comparable, nor combinable given the problem of false discovery rates (FDR) that pile up when dataset results are combined.
For example, if we combine 3 datasets that were each filtered to 1% FDR, the maximum FDR of the combined dataset is now 3% because it is unlikely that the random decoy hits are shared across each dataset. 
To address this, in 2005 the PeptideAtlas concept was started to ingest as many publically available datasets as possible per organism, search the data through a single pipeline in total and arrive at a datasets with a total controlled 1% FDR at the protein level [@PMID:16052627; @PMID:16381952].
The PeptideAtlas website (www.peptideatlas.org) is a therefore a multi-organism, publicly accessible compendium of peptides identified in large sets of tandem mass spectrometry proteomics experiments.
Mass spectrometer output files are collected for human, mouse, yeast, and many other organisms of research interest, and searched using the latest search engines and genome derived protein sequences.
All results of sequence and spectral library searching on PeptideAtlas are processed through the Trans Proteomic Pipeline to derive a probability of correct identification for all results in a uniform manner to insure a high quality database, along with false discovery rates at the whole atlas level.

The most recognizable MS data compendium is the Human PeptideAtlas which is produced yearly since 2005 to derive all the peptide sequence knowledge of the current human proteome (Figure 17A). 
As of 2023, the Human PeptideAtlas contains the knowledge of over 93% of the human proteome, with over 189K MS runs and 3.6B spectra searched resulting in 3.4m peptides identified and 17,245 proteins identified from the 19,600 total proteins possible.
The number of proteins has been incrementally increases year over year as new public data becomes available (Figure 17B).

![**The Human PeptideAtlas as of 2023.**
A) The current total search space and identified elements of the 2023 Human PeptideAtlas. 
B) Historical cumulative plot of the identified total proteins (blue vertical bars) and the unique proteins identified per dataset (red vertical bars) over the total period of 2005-2023.
](images/peptide_atlas.jpg){#fig:peptide-atlas tag="17" width="100%"}

For the presentation of selected reaction monitoring (SRM) targeted peptide assays, there are two components of the PeptideAtlas ecosystem where the PeptideAtlas SRM Experiment Library (PASSEL) is presented to enable submission, dissemination, and reuse of SRM experimental results from analysis of biological samples [@PMID:22318887; @PMID:24939129]. 
The PASSEL system acts as a data repository by allowing researchers with SRM data to deposit their data in parallel with journal publication. and other users can search existing data to obtain the parameters for replication in their own laboratory. 
Another unique component for SRM data repositories is the SRMAtlas website, which provides definitive coordinates for all possible proteins within an organism to conduct targeted SRM assays that conclusively identify the respective peptide in biological samples. 
As an example, the Human SRMAtlas provides data on 166,174 synthetic proteotypic human peptides, providing multiple, independent assays to quantify any human protein and numerous spliced variants, non-synonymous mutations, and post-translational modifications [@PMID:27453469]. 
The data are freely accessible as a resource at http://www.srmatlas.org/.

### Other knowledge bases
There are many more proteomics knowledgebases, such as PRIDE [@DOI:10.1093/nar/gkj138; @DOI:10.1093/nar/gkab1038], the proteomics standards initiative (PSI) [@DOI:10.1021/acs.jproteome.7b00370], Massive [@PMID:30172843], Proteometools [@DOI:10.1038/nmeth.4153], Panorama [@DOI:10.1021/pr5006636], Chorus [@URL:https://chorusproject.org/pages/index.html], and iProX [@DOI:10.1093/nar/gky869].
